src/query/functions/src/scalars/datetime.rs

```diff
@@ -108,6 +109,9 @@ pub fn register(registry: &mut FunctionRegistry) {
 
     // [date | timestamp] +/- number
     register_timestamp_add_sub(registry);
+
+    // convert_timezone( target_timezone, date, src_timezone)
```

Comment:
The arguments are in the wrong order. They should be `src_timezone, target_timezone, source_timestamp`

Reply:
Yes.

Need support 

CONVERT_TIMEZONE( <source_tz> , <target_tz> , <source_timestamp_ntz> )


and 


CONVERT_TIMEZONE( <target_tz> , <source_timestamp> )